Changes

```diff
@@ -1,5 +1,7 @@
 LIST OF CHANGES FOR NPG-QC PACKAGE
 
+ - creating new SQL tables to handle usability feature
```

Comment:
use utility for everything related to this project so there is a single term everywhere